src/translate.py

```diff
@@ -733,11 +733,23 @@ def f64(left: Expression, op: str, right: Expression) -> "BinaryOp":
             floating=True,
         )
 
-    def is_boolean(self) -> bool:
```

Comment:
`&&` and `||` are "boolean", but the existing logic only cared about comparisons.